OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -132,6 +447,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">10</dim>
                                         <dim type="left">9</dim>
                                     </dimensions>
+                                    <note>The characters per line are 13-16.</note>
+                                    <ab type="ruling" subtype="pattern">
+                                        <locus from="6ra" to="53vb"/>    
+                                        Ruling pattern:
+                                        1A-1A-1A1A/0-0/0-0/C.</ab>
```

Comment:
ruling pattern correct?

Reply:
The ruling is the pattern for two columns throughout 6ra-53vb, but on the last folia the text is written in one column on this ruling pattern for two columns.